hack/install.sh

```diff
@@ -102,7 +108,10 @@ rpm_import_repository_key() {
 	local key=$1; shift
 	local tmpdir=$(mktemp -d)
 	chmod 600 "$tmpdir"
-	gpg --homedir "$tmpdir" --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" || gpg --homedir "$tmpdir" --keyserver pgp.mit.edu --recv-keys "$key"
+	for key_server in $key_servers ; do
+		gpg --homedir "$tmpdir" --keyserver "$key_server" --recv-keys "$key" && break
+	done
+	gpg --homedir "$tmpdir" -k "$key" >/dev/null
```

Comment:
Just want to confirm that @estesp and @tianon are correct. 2 different mechanisms that load keys (for RPM and DEB) and both get a validation command after that helps the script fail properly.

Thanks for having a look, folks!